6.0/m-cluster/README.md

```diff
@@ -0,0 +1,22 @@
+# m-cluster (CDM) Sample Config
+
+### What is in this config repository?
+AM and IDM config repos that get mounted under /git on each pod via
+the charts 
+
+### Common Config
+- Prometheus endpoints exposed
+- Logging to stdout
+- Tuned for m-cluster
+
+
+### AM Config
+- Single realm
+- OAuth 2.0 Server and Client
+- Separate userstore, configstore and ctsstore
+- LDAP Auth module default
+
+### IDM Config
+- Postgres as repo
+- Syncronization to LDAP
```

Comment:
s/Syncronization/Synchronization/ (typo)

Reply:
will fix in the next PR